modules/administration/pages/auth-methods.adoc

```diff
@@ -2,99 +2,102 @@
 = Authentication Methods
 
 
+
 == Authentication Via PAM
 
 
-As security measures become increasingly complex, SUSE Manager supports network-based authentication systems via Pluggable Authentication Modules (PAM). PAM is a suite of libraries that allows to integrate SUSE Manager with a centralized authentication mechanism, thus eliminating the need to remember multiple passwords.
-SUSE Manager supports LDAP, Kerberos, and other network-based authentication systems via PAM.
-To enable SUSE Manager to use PAM in your organization's authentication infrastructure, follow the steps below.
+As security measures become increasingly complex, {productname} supports network-based authentication systems via Pluggable Authentication Modules (PAM).
+PAM is a suite of libraries that allows to integrate {productname} with a centralized authentication mechanism, thus eliminating the need to remember multiple passwords.
+{productname} supports LDAP, Kerberos, and other network-based authentication systems via PAM.
+
+
 
+=== Enabling PAM
 
-. Set up a PAM service file (default location: [path]``/etc/pam.d/susemanager`` ) then enforce its use by adding the following line to [path]``/etc/rhn/rhn.conf`` :
+To enable {productname} to use PAM in your organization's authentication infrastructure, follow the steps below.
+
+
+. Set up a PAM service file (default location: [path]``/etc/pam.d/susemanager``).
+Enforce its use by adding the following line to [path]``/etc/rhn/rhn.conf``:
 +
 
 ----
 pam_auth_service = susemanager
 ----
 +
-NOTE: This assumes the PAM service file is named susemanager.
+NOTE: This assumes the PAM service file is named [path]``susemanager``.
 +
 
-. To enable a new or existing user to authenticate with PAM, proceed to the menu:Create User[] page and select the checkbox labeled Pluggable Authentication Modules (PAM) positioned below the password and password confirmation fields.
-. To authenticate a {slsa} system against Kerberos add the following lines to [path]``/etc/pam.d/susemanager`` :
+. Navigate to the menu:Create User[] page to enable a new or existing user to authenticate with PAM
```

Comment:
```suggestion
. In the {productname} {webui}, navigate to menu:Create User[] and enable a new or existing user to authenticate with PAM
```

This could probably be clearer.